docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.8"
+
+services:
+    localstack:
+        container_name: "${LOCALSTACK_DOCKER_NAME:-localstack-main}"
+        image: localstack/localstack-pro:latest # required for Pro
+        pull_policy: always
+        ports:
+            - "127.0.0.1:4566:4566" # LocalStack Gateway
+            - "127.0.0.1:4510-4559:4510-4559" # external services port range
+            - "127.0.0.1:443:443" # LocalStack HTTPS Gateway (Pro)
+        environment:
+            # Activate LocalStack Pro: https://docs.localstack.cloud/getting-started/auth-token/
+            - LOCALSTACK_AUTH_TOKEN=${LOCALSTACK_AUTH_TOKEN:?} # required for Pro
+            - ENABLE_DMS=1
+        volumes:
+            - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
+            - "/var/run/docker.sock:/var/run/docker.sock"
+        healthcheck:
+            test: curl --fail localhost.localstack.cloud:4566/_localstack/health || exit 1
+            interval: 5s
+            timeout: 2s
+            retries: 5
+            start_period: 10s
+    postgres_server:
+        container_name: dms-sample-postgres
+        image: postgres
+        ports:
+            - "127.0.0.1:5432:5432"
+        restart: always
+        command: -c 'wal_level=logical' -c 'max_replication_slots=10' -c 'max_wal_senders=10'
+        environment:
+            - POSTGRES_DB=${DB_NAME:-testdb}
+            - POSTGRES_USER=${USERNAME:-admin123}
+            - POSTGRES_PASSWORD=${USERPWD:-mysecretpassword}
```

Comment:
question: Why do we spawn another external Postgres server? It seems this Stack is not making use of this external PG, because we always spawn an RDS Cluster, and always connect to it. We can probably remove this, and remove everything related to it (the environment vars in the Makefile)

Reply:
Good catch! This was to have an external postgres server but since this is not mentioned or used yet in the sample we can remove it. Thanks 🙌